modules/core/src/test/java/io/fluo/recipes/transaction/RecordingTransactionIT.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2015 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.recipes.transaction;
+
+import java.io.File;
+
+import io.fluo.api.client.FluoClient;
+import io.fluo.api.client.FluoFactory;
+import io.fluo.api.config.FluoConfiguration;
+import io.fluo.api.data.Bytes;
+import io.fluo.api.data.Column;
+import io.fluo.api.mini.MiniFluo;
+import io.fluo.api.types.StringEncoder;
+import io.fluo.api.types.TypeLayer;
+import io.fluo.api.types.TypedTransaction;
+import io.fluo.recipes.serialization.SimpleSerializer;
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RecordingTransactionIT {
+
+  private MiniFluo miniFluo;
+
+  @Before
+  public void setUpFluo() throws Exception {
+    FileUtils.deleteQuietly(new File("target/mini"));
+    FluoConfiguration props = new FluoConfiguration();
+    props.setApplicationName("recordtx");
+    props.setMiniDataDir("target/mini");
+    miniFluo = FluoFactory.newMiniFluo(props);
+  }
+
+  @After
+  public void tearDownFluo() throws Exception {
+    if (miniFluo != null) {
+      miniFluo.close();
+    }
+  }
+
+  @Test
+  public void testBasic() {
+    try (FluoClient client = FluoFactory.newClient(miniFluo.getClientConfiguration())) {
+      RecordingTransaction rtx = RecordingTransaction.wrap(client.newTransaction());
```

Comment:
this could be made into a unit test.. could use EasyMock to create a Transaction object from the interface.... or could create a DevNullTransaction class that extends Transaction and does nothing...